app/renderer/actions/Session.js

```diff
@@ -250,6 +251,13 @@ export function newSession (caps, attachSessId = null) {
           return;
         }
         https = false;
+        if (!desiredCapabilities['sauce:options']) {
+          desiredCapabilities['sauce:options'] = {};
+        }
+        if (!desiredCapabilities['sauce:options'].name) {
+          const dateTime = moment().format('MMM DD -- h:mma');
```

Comment:
would it be possible to format the string according to active user time format settings?

Reply:
@mykola-mokhnach could you be specific, I'm not familiar with what that is.

Reply:
check `Multiple Locale Support` at https://momentjs.com/